client/my-sites/plan-features/index.jsx

```diff
@@ -468,6 +483,7 @@ export default connect(
 				const newPlan = isNew( plan ) && ! isPaid;
 				const currentPlan = sitePlan && sitePlan.product_slug;
 				const showMonthlyPrice = ! relatedMonthlyPlan && showMonthly;
+				const currentPlanMatch = !! ( sitePlan && ( getPlanClass( sitePlan.product_slug ) === getPlanClass( plan ) ) );
```

Comment:
I'd expect to have something similar to what we have in `lib/products-values`, but working with plan slugs instead of plan objects. My main rationale for preferring that way is that I want to keep the plan slug logic on as less places as possible, because this way we make our lives easier for future changes (when removing, adding, or even renaming plans). The more we bloat the codebase with raw plan slug imports use them within logic, the more hard time we'll have when maintaining the codebase in the future.